lms/templates/courseware/info.html

```diff
@@ -58,8 +58,9 @@ <h2 class="title">${_("You are not enrolled yet")}</h2>
     <div class="container">
       <div class="home">
         <div class="page-header-main">
-            <h1 class="page-title">${_("Welcome to {org}'s {course_name}!").format(org=course.display_org_with_default, course_name=course.display_number_with_default)}</h1>
-            <h2 class="page-subtitle">${course.display_name_with_default}</h2>
+            <h3 class="page-title">${_("Welcome to {org}'s {course_name}!").format(org=course.display_org_with_default, course_name=course.display_number_with_default)}
+            <div class="page-subtitle">${course.display_name_with_default}</div>
```

Comment:
@andy-armstrong A heading should never be followed immediately by another heading.  Headings introduce/summarize content.  It's a common misconception to markup subheadings this way.  The W3C addresses this directly here:
https://www.w3.org/TR/html5/common-idioms.html#sub-head

We basically have two options:

1. include the "subheading" with the heading as @luckyv has done here, OR
2. make it a paragraph that follows the heading.

It seems important enough to me here, to be included with the heading.